docs/frontend/overview.rst

```diff
@@ -0,0 +1,172 @@
+Overview
+========
+
+The adhocracy 3 frontend is an opinionated web framework based on
```

Comment:
as mentioned in the chat, if we document a3 as a framework, then it is our responsability that external developers can use what we write to develop their participation processes and get support if they find bugs in our software. It is very different from our actual responsabilities.